source/containers/document-pii-detection/main.py

```diff
@@ -195,18 +201,93 @@ def create_glue_table(glue_client, database_name, table_name, glue_table_info):
             TableInput=glue_table_info
         )
 
-    print(response)
+
+# Worker function to process a detection_files item
+def worker(queue,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This function calls 16 other functions. By comparison, 98% of the functions in the CodeGuru reference dataset use fewer. This indicates the function is highly coupled with other functions. A function that is highly coupled with other functions is difficult to understand and its behavior might change unexpectedly when one of its referenced functions is updated. High coupling could also increase the integration test complexity, maintenance cost and technical debt. We recommend that you simplify this function or break it into multiple functions.